src/robot_localization_estimator.cpp

```diff
@@ -319,9 +202,8 @@ void RobotLocalizationEstimator::interpolate(const EstimatorState& given_state_1
                                              EstimatorState& state_at_req_time) const
 {
   /*
-   * TODO: Right now, we only extrapolate from the last known state before the requested time.
-   * But as the state after the requested time is also known, we may want to perform
-   * interpolation between states.
+   * TODO: Right now, we only extrapolate from the last known state before the requested time. But as the state after
```

Comment:
This todo is still open. The state interpolation can be done using state estimates on both sides of the requested time as constraints, but I have no knowledge about interpolation of the covariance matrices. By the way, interpolation does not take into account the system model from the filter, so extrapolation from the nearest known time might actually be a nicer option.